tests/migrations/test_operations.py

```diff
@@ -168,6 +148,24 @@ def set_up_test_model(
 
         return self.apply_operations(app_label, ProjectState(), operations)
 
+    def cleanup_test_model(self):
+        table_names = frozenset(connection.introspection.table_names()) - self._base_table_names
+        with connection.schema_editor() as editor:
+            with connection.constraint_checks_disabled():
+                for table_name in table_names:
+                    editor.execute(editor.sql_delete_table % {
+                        'table': editor.quote_name(table_name),
+                    })
+
+    @classmethod
```

Comment:
I would expect to see the setup/teardown methods at the top of the class.

Reply:
moved